src/article-editor.jsx

```diff
@@ -66,7 +67,7 @@ var ArticleEditor = React.createClass({
 
     render: function() {
 
-        return <div className="framework-perseus perseus-article">
+        return <div id="perseus" className="framework-perseus perseus-article">
```

Comment:
Okay :). We should fix any css that relies on `#perseus` to use `.framework-perseus` (after checking that works okay where we render other uses of perseus). ids are evil!